### ISSUE TITLE
Add GUI for color adjustment (contrast, saturation etc.)

### ISSUE THREAD

**Issue by haasn** [tomasklaen/uosc#616]
This would honestly be amazingly useful, I hate having to use keybinds to adjust these parameters. Probably just add it to the default context menu.

**Comment by po5** [comment]
Not sure that would be very useful with the default menu curtain obscuring the image.

**Comment by christoph-heinrich** [comment]
Maybe we could generalize the slider from the volume slider and make that usable from the controls or something.
But that would make the controls syntax even more complicated.

**Comment by tomasklaen** [comment]
Yeah would be nice to have. No promises or ETAs though. I'm already neglecting uosc quite a bit, and dunno how soon it'll change :(

**Comment by gunir** [comment]
If someone is interested in this feature feel free to create PR and it's also not too bad to use 1,2,3,4 of mpv to change brightness and contrast, which is not too bad in fact using hotkey feel very easy for me in this case.

**Comment by mrfragger** [comment]
I just add two 4 shortcut keys but really only use the little menus.  I like doing 144p on youtube and black and white it and knocking down contrast by -50 to focus on subs.

<img width="434" alt="image" src="https://github.com/user-attachments/assets/18d9acae-7ce7-4e65-a670-a68f4f47b1de">

```
Ctrl+Alt+z add contrast -25                   #! [Video] > [Display] > Dark Mode
Ctrl+Alt+x add contrast 25                  #! [Video] > [Display] > Light Mode
Ctrl+Alt+Z add saturation -100                    #! [Video] > [Display] > B&W video
Ctrl+Alt+X add saturation 100                    #! [Video] > [Display] > B&W video off
```

then to add the four buttons in the uosc timeline

`command:dark_mode:add contrast -25?Ctrl+Alt+z Dark mode,command:light_mode:add contrast 25?Ctrl+Alt+x Light mode,command:invert_colors:add saturation -100?Ctrl+Alt+Z B&W video,command:invert_colors_off:add saturation 100?Ctrl+Alt+X B&W off`
